web_src/js/components/DiffFileTreeItem.vue

```diff
@@ -20,7 +20,7 @@
       />
 
       <!-- Directories -->
-      <div v-if="!item.isFile" class="directory gt-p-1" @click.stop="handleClick(item.isFile)">
+      <div v-if="!item.isFile" class="directory gt-p-1 gt-ac" @click.stop="handleClick(item.isFile)">
```

Comment:
(as above)

Reply:
Can not put into `.filewrapper` CSS because directories have no hover effect, only files.

Reply:
But I guess I can do a bit of refactoring here anyways...

Reply:
Actually, no, I don't think it's worth the effort trying to do this with CSS. It's be a net increase in code and would likely require a new classname or new CSS rule that matches both.